src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/CustomDescription.tsx

```diff
@@ -45,6 +48,30 @@ const TxInfo = styled.div`
   padding: 8px 8px 8px 16px;
 `
 
+const TxInfoDetails = ({ data }: { data: DataDecoded }): React.ReactElement => {
+  const methodName = data?.method ? ` (${data.method})` : ''
+
```

Comment:
is it possible to have arguments if there is no method?

Reply:
nope! actually that check is unnecessary.

After this review, did some fixes.

Thanks!